### ISSUE TITLE
Errors in Grafana Dashboards after manual upgrade

### ISSUE THREAD

**Issue by SigLinJo** [adriankumpf/teslamate#210]
Hi,

I updated a while ago to v1.11.0, had some issues with running the database migration due to having missed to make the database user superuser, fixed it and the migration reported successful.

Teslamate seems to be running normally but I have issues in Grafana.

Drives dashboard just has a red left top corner and the error message: pq: column start_geofence.address_id does not exist

![image](https://user-images.githubusercontent.com/19478141/67147748-f079ac00-f297-11e9-8ef7-17b03b4c6228.png)

Locations has this error: pq: column g.address_id does not exist

![image](https://user-images.githubusercontent.com/19478141/67147765-333b8400-f298-11e9-8a85-00b22ed4e792.png)

Carges also reports: pq: column g.address_id does not exist

Charging stats has this error: pq: column geofence.address_id does not exist

![image](https://user-images.githubusercontent.com/19478141/67147975-5f580480-f29a-11e9-9f46-e74b604c78e4.png)

Locations shows no top destinations with this error: pq: column g.address_id does not exist

All other dashboards seem to work normally. Does anyone have any idea of what I have done wrong?

I run on FreeBSD 11.2 in a jail on FreeNas.

Grafana is version 6.3.5, updated at the same time as teslamate.
I run postgresql96-server-9.6.15, might have updated that also at the same time but I don't know.

Thankful for any help I can get! I don't want to lose all the data I have collected this far by starting over...

Best Regards
Sigfrid

**Comment by adriankumpf** [comment]
Your dashboards are outdated. Re-import them as described here: https://github.com/adriankumpf/teslamate/wiki/Manual-Installation#importing-dashboards

**Comment by SigLinJo** [comment]
Ah, thought I had done that, tried again and got this output in the shell:

Output:
{"id":14,"slug":"charge-details","status":"success","uid":"BHhxFeZRz","url":"/d/BHhxFeZRz/charge-details","version":8}
✔ Dashboard charge-details exported successfully.
Output:
{"id":13,"slug":"charges","status":"success","uid":"TSmNYvRRk","url":"/d/TSmNYvRRk/charges","version":9}
✔ Dashboard charges exported successfully.
Output:
{"id":12,"slug":"charging-stats","status":"success","uid":"-pkIkhmRz","url":"/d/-pkIkhmRz/charging-stats","version":8}
✔ Dashboard charging-stats exported successfully.
Output:
{"id":11,"slug":"degradation","status":"success","uid":"riqUfXgRz","url":"/d/riqUfXgRz/degradation","version":8}
✔ Dashboard degradation exported successfully.
Output:
{"id":10,"slug":"drive-details","status":"success","uid":"zm7wN6Zgz","url":"/d/zm7wN6Zgz/drive-details","version":9}
✔ Dashboard drive-details exported successfully.
Output:
{"id":9,"slug":"drives","status":"success","uid":"Y8upc6ZRk","url":"/d/Y8upc6ZRk/drives","version":9}
✔ Dashboard drives exported successfully.
Output:
{"id":8,"slug":"efficiency","status":"success","uid":"fu4SiQgWz","url":"/d/fu4SiQgWz/efficiency","version":9}
✔ Dashboard efficiency exported successfully.
Output:
{"id":7,"slug":"locations","status":"success","uid":"ZzhF-aRWz","url":"/d/ZzhF-aRWz/locations","version":8}
✔ Dashboard locations exported successfully.
Output:
{"id":6,"slug":"mileage","status":"success","uid":"NjtMTFggz","url":"/d/NjtMTFggz/mileage","version":8}
✔ Dashboard mileage exported successfully.
Output:
{"id":5,"slug":"position","status":"success","uid":"Dz6UUdzWk","url":"/d/Dz6UUdzWk/position","version":8}
✔ Dashboard position exported successfully.
Output:
{"id":4,"slug":"states","status":"success","uid":"xo4BNRkZz","url":"/d/xo4BNRkZz/states","version":8}
✔ Dashboard states exported successfully.
Output:
{"id":3,"slug":"updates","status":"success","uid":"IiC07mgWz","url":"/d/IiC07mgWz/updates","version":8}
✔ Dashboard updates exported successfully.
Output:
{"id":2,"slug":"vampire-drain","status":"success","uid":"zhHx2Fggk","url":"/d/zhHx2Fggk/vampire-drain","version":8}
✔ Dashboard vampire-drain exported successfully.
Output:
{"id":1,"slug":"visited","status":"success","uid":"RG_DxSmgk","url":"/d/RG_DxSmgk/visited","version":8}
✔ Dashboard visited exported successfully.
✔ 14 dashboards exported successfully.

Still not working...

Tried to manually import them and it worked. Something in the automated import did not work...

Found it! The automatic import is still looking in the wrong place, found my outdated files that I had copied last time and therefore updated the dashboards with outdated versions. Removed my copy of dashboards folder and automatic import dont work anymore...

Best regards
Sigfrid
